amivapi/tests/users/test_user_cleanup.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Tests for user module.
+
+Test if inactive non-members get deleted from database
+"""
+
+from datetime import datetime, timedelta
+from lib2to3.pgen2 import token
+from signal import SIG_DFL
+from freezegun import freeze_time
+
+from amivapi import cron
+from amivapi.tests import utils
+from amivapi.users import cleanup
+
+
+
+class UserCleanupTest(utils.WebTest):
+    """Test automatic User Cleanup"""
+
+    def test_autodelete(self):
+        entries = [
```

Comment:
This array is quiet large. We could make is smaller as follows. With this, we just see the relevant information and also see the updated timestamp intended for the user.

```python
cleanup_time = "2019-12-24T14:15:22Z"

entries = [
    {
        # Non-member updated 6 months prior to cleanup.
        'membership': 'none',
        'updated': '2019-08-24T14:15:22Z'
    },
    {
        # Regular member updated 6 months prior to cleanup.
        'membership': 'regular',
        'updated': '2019-08-24T14:15:22Z'
    },
    {
        # Non-member updated 18 months prior to cleanup.
        # This use should be removed during cleanup.
        'membership': 'none',
        'updated': '2018-08-24T14:15:22Z'
    },
    {
        # Honorary member updated 18 months prior to cleanup.
        'membership': 'honorary',
        'updated': '2018-08-24T14:15:22Z'
    }
]
```